src/pages/registeredPage.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import {Link, useParams}
+    from "react-router-dom";
+import background from '../img/confetee.png'
+
+function RegisteredPage() {
+    const {id} = useParams();
+
+    return (
+        <div style={{
+            backgroundImage: `url(${background})`,
+            backgroundPosition: 'center',
```

Comment:
can it be a css classname instead inlined styles?
```suggestion
        <div style={{
            backgroundImage: `url(${background})`,
            backgroundPosition: 'center',
```

Reply:
added classes